### PR TITLE
More API client examples in the dev documentation

### PR DESCRIPTION
**Pull request by jaegeral**
Mention the addition of:

- add single events to a sketch
- add tags to events

### CODE REVIEW

**Review by obsidianforensics — APPROVED** [review]

File: docs/developers/api-client.md [hunk 1]
```diff
@@ -498,12 +498,78 @@ The output can be:
 + A pandas DataFrame if the `as_pandas=True` is set
 + A python dict (default behavior)
 
-### Other Options
+
+## Add data
+
+#### Manually add events to the sketch
+
+Fill in your event data Put in your message, date and timestamp description
```
Comment (berggren) [inline docs/developers/api-client.md:506]:
```suggestion
Fill in your event data, you need at least these fields: message, date and timestamp description
```

File: docs/developers/api-client.md [hunk 2]
```diff
@@ -498,12 +498,78 @@ The output can be:
...
+
+
+```python
+
+message = "foobar"
+date = "2020-08-06T12:48:06.994188Z"
+timestamp_desc = "Test_description"
+
+#attributes: A dict of extra attributes to add to the event.
```
Comment (berggren) [inline docs/developers/api-client.md:515]:
```suggestion
# Attributes: A dict of extra attributes to add to the event.  
```

File: docs/developers/api-client.md [hunk 3]
```diff
@@ -498,12 +498,78 @@ The output can be:
...
+```python
+
+message = "foobar"
+date = "2020-08-06T12:48:06.994188Z"
+timestamp_desc = "Test_description"
+
+#attributes: A dict of extra attributes to add to the event.  
+attributes = {"a": "alpha", "o": "omega", "g": "gamma"}
+
+#tags: A list of strings to include as tags.
```
Comment (berggren) [inline docs/developers/api-client.md:518]:
```suggestion
# Tags: A list of strings to include as tags.
```

**Review by berggren — APPROVED** [review]
A couple of suggestions, otherwise LGTM

**Pull request merged**